src/components/lib/FlowGrid/FlowGrid.jsx

```diff
@@ -69,8 +69,8 @@ export default class FlowGrid extends Component{
   }
 
   _columnCount() {
-    const lowestItem = Math.max(...this.props.items.map(e => parseInt(e.location.column))) + 3
-    const selected = parseInt(this.props.selected.column) + 3
+    const lowestItem = Math.max(...this.props.items.map(e => parseInt(e.location.column))) + 4
+    const selected = parseInt(this.props.selected.column) + 4
     return Math.max(6, lowestItem, selected) || 6;
```

Comment:
Might be a bit cleaner to put the `+ 4`s in the return. e.g.

return Math.max(2, lowestItem, selected) + 4

Also, I don't think you need the || 6. And if you do you could put it in the map which would be better as then a single `NaN` wouldn't implode the whole calculation (potentially giving too few columns) (e.g. `...(e => parseInt(e.location.cloumn) || 0`)